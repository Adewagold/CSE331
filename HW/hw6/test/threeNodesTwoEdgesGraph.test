# Tests build threeNodesTwoEdgesGraph

# Load threeNodesTwoEdgesGraph.tsv
LoadGraph three_nodes_two_edges_graph threeNodesTwoEdgesGraph.tsv

# Check if build graph is correct
ListNodes three_nodes_two_edges_graph
ListChildren three_nodes_two_edges_graph node1
ListChildren three_nodes_two_edges_graph node2
ListChildren three_nodes_two_edges_graph node3