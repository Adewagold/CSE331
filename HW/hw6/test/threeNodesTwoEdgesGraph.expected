# Tests build threeNodesTwoEdgesGraph

# Load threeNodesTwoEdgesGraph.tsv
loaded graph three_nodes_two_edges_graph

# Check if build graph is correct
three_nodes_two_edges_graph contains: node1 node2 node3
the children of node1 in three_nodes_two_edges_graph are: node2(1.000)
the children of node2 in three_nodes_two_edges_graph are: node1(1.000)
the children of node3 in three_nodes_two_edges_graph are: